jenkins/ci.suse.de/cloud-ardana9.yaml

```diff
@@ -139,7 +139,7 @@
       ci,smoke,smoke-upstream,defcore,full,barbican,compute,designate,identity,lbaas,\
       magnum,manila,monasca,network,neutron-api,swift"
     qa_test_list: "\
-      iverify,ceilometer,ceilometer_capacity_management,cinder,cinder-parallel,getput,\
+      iverify,monasca-ceilometer,ceilometer_capacity_management,cinder,cinder-parallel,getput,\
```

Comment:
Was removal of ceilometer intentional?

Reply:
Ceilometer API is removed from C9, so ceilometer CLI testing is invalid now